platforms/atsam4s/api/dac.c

```diff
@@ -1,6 +1,57 @@
 #include "libflipper.h"
-#include <dacc.h>
+#include <dacc.h> 
+#include "sam4s16b.h.h"
 
 LF_FUNC int dac_configure(void) {
-    return lf_success;
+
+    // 12 bit DAC
+    // initialize system clock 
+    sysclk_init()
+
+    // Enable the Peripheral clock 
+    // DAC Peripheral ID = 30 
+    sysclk_enable_peripheral_clock(30)
+
+    // Reset the DAC 
+    dacc_reset(DACC)
+
+    // Set DAC Transfer Mode
+    dacc_set_transfer_mode(DACC, DACC_MR_WORD)
+
+    // Set the timing for the DAC
+    dacc_set_timing(DACC, DACC_MR_REFRESH_Pos, DACC_MR_MAXS_NORMAL, DACC_MR_STARTUP_24)
+
+    // set Digital to Analog Conversion channel to channel 1
+    dacc_set_channel_selection(DACC, DAC0); 
```

Comment:
This should be be done in another function, or passed as an argument. We don't want to turn on all the channels by default.